components/script/script_module.rs

```diff
@@ -871,19 +872,22 @@ pub(crate) enum ModuleOwner {
     #[allow(dead_code)]
     Worker(TrustedWorkerAddress),
     Window(Trusted<HTMLScriptElement>),
+    DynamicModule(Trusted<DynamicModuleOwner>),
 }
 
 impl ModuleOwner {
     pub fn global(&self) -> DomRoot<GlobalScope> {
         match &self {
             ModuleOwner::Worker(worker) => (*worker.root().clone()).global(),
             ModuleOwner::Window(script) => (*script.root()).global(),
+            ModuleOwner::DynamicModule(dynamic_module) => (*dynamic_module.root()).global(),
         }
     }
 
     pub fn notify_owner_to_finish(&self, module_identity: ModuleIdentity) {
         match &self {
             ModuleOwner::Worker(_) => unimplemented!(),
+            ModuleOwner::DynamicModule(_) => unimplemented!(),
```

Comment:
Currently, we will register `finish_dynamic_module` with `append_dynamic_module_handler` which is different from the normal `notify_owner_to_finish` flow so this is `unimplemented` for now.

But, I will want to refactor it to be a _trait_ and hold a `notify_to_finish` function so that we can just call it here and then I can try to remove the `append_dynamic_module_handler` function to remove more duplicated-like code!